pkg/scannerv4/mappers/mappers.go

```diff
@@ -77,6 +78,14 @@ var (
 	}
 )
 
+type epssDetail struct {
+	ModelVersion string  `json:"ModelVersion"`
+	CVE          string  `json:"CVE"`
+	Date         string  `json:"Date"` // Use string as there's no need to manipulate the Date as a time.Time object
+	EPSS         float64 `json:"EPSS"`
+	Percentile   float64 `json:"Percentile"`
+}
```

Comment:
Isn't there already `epss.EPSSItem` in Claircore? We can just use that instead of making a new struct